app/Helpers/GendersHelper.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Helpers;
+
+class GendersHelper
+{
+    /**
+     * Return a collection of genders.
+     *
+     * @return \Illuminate\Support\Collection
+     */
+    public static function getGendersInput()
+    {
+        $genders = auth()->user()->account->genders->map(function ($gender) {
+            return [
+                'id' => $gender->id,
+                'name' => $gender->name,
+            ];
+        });
+        $genders = CollectionHelper::sortByCollator($genders, 'name');
+        $genders->prepend(['id' => '', 'name' => '']);
```

Comment:
I don't like that very much. Instead of providing a blank field, the choice should be `No gender`. Otherwise I think the user is confused.

Reply:
Yes, I've changed it